presto-main/src/main/java/com/facebook/presto/operator/SetBuilderOperator.java

```diff
@@ -189,12 +193,30 @@ public boolean isFinished()
         return finished;
     }
 
+    /**
+     * This function processes unfinished work for the current page, and returns if it still
```

Comment:
Let's remove all the javadoc in this commit given they are self-explanatory.

Reply:
ok